tools/SourceKit/tools/sourcekitd/lib/API/sourcekitdAPI-InProc.cpp

```diff
@@ -237,19 +237,15 @@ class SKDBool: public SKDObject {
 
 class SKDCustomData: public SKDObject {
 public:
-  SKDCustomData(CustomBufferKind BufferKind, 
-                std::unique_ptr<llvm::MemoryBuffer>& MemBuf)
-  : SKDObject(ObjectKind::CustomData), BufferKind(BufferKind),
-    BufferPtr(llvm::MemoryBuffer::getMemBufferCopy(
-                                                MemBuf->getBuffer(), 
-                                                MemBuf->getBufferIdentifier())) 
+  SKDCustomData(std::unique_ptr<llvm::MemoryBuffer>& MemBuf)
```

Comment:
You are changing this from copying the buffer to taking ownership of it, so the type should be rvalue reference (`&&`) or else by value, not by lvalue reference.